iree_tests/conftest.py

```diff
@@ -292,6 +295,8 @@ def __init__(self, spec, **kwargs):
         self.run_args.extend(self.spec.iree_run_module_flags)
         self.run_args.append(f"--flagfile={self.spec.data_flagfile_name}")
 
+        self.benchmark_args = ["iree-benchmark-module", "--device=local-task", f"--module={prompt_encoder_path}/model_sdxl_cpu_llvm_task_real_weights.vmfb", f"--parameters=model={prompt_encoder_path}/real_weights.irpa", f"--module={scheduled_unet_path}/model_sdxl_cpu_llvm_task_real_weights.vmfb", f"--parameters=model={scheduled_unet_path}/real_weights.irpa", f"--module={vae_decode_path}/model_sdxl_cpu_llvm_task_real_weights.vmfb", f"--parameters=model={vae_decode_path}/real_weights.irpa", f"--module={vmfb_name}", "--function=tokens_to_image", "--input=1x4x128x128xf16", "--input=1xf16", "--input=1x64xi64", "--input=1x64xi64", "--input=1x64xi64", "--input=1x64xi64"]
```

Comment:
Looking at this with fresh eyes... if all you want for the moment is to benchmark a single program, you can just script that instead of hooking into pytest.

benchmark_sdxl.sh:
```bash
# license header

set -xeuo pipefail

iree-compile ...
iree-benchmark-module ...
```

We should not be hardcoding any paths or arguments into the test runner.

Reply:
This is definitely what should be done. Thanks :) Do you know if we run a script in CI if it shows the output of running that script?

Reply:
`set -xeuo pipefail` helps with that. I can never remember the individual flags, so 
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

* `x`: print commands as they are run
* `e`: stop immediately if any command fails (instead of just continuing... yes, bash is silly)
* `u`: treat unset variables as an error (instead of just continuing with them unset...)
* `-o pipefail`: exit the script with the return code of the last command (instead of silently succeeding if the script fails)

So you should see a mix of
```
command
output of command
command
output of command
```